js/client-formatmods.js

```diff
@@ -0,0 +1,49 @@
+(function(exports, $){

```

Comment:
This should be `(function (exports, $){`

Which is the first of around 30 things Travis CI is currently telling you is wrong. Do you know how to use `npm test`? That will give you the same errors Travis CI is giving, without needing to manually push and check this page.